doc/apps/voila.md

```diff
@@ -1,17 +1,26 @@
 # Voilà
 
-```{tip}
-If you haven't deployed your first application, check out the [quickstart guide.](../quickstart/app.md)
+## Deploy an example
+
+To download and deploy an example Voila application simply run:
+
+```sh
+pip install ploomber-cloud
+ploomber-cloud key YOUR-KEY
+ploomber-cloud examples voila/chat-with-csv
+cd basic-app
+ploomber-cloud init
+ploomber-cloud deploy --watch
```

Comment:
let's split this in multiple snippets and provide more context on what the user should expect. for commands with short outputs, we can put a sample output. for others, we can just describe what they should expect

Reply:
[Split up and added more context.](https://github.com/ploomber/doc/pull/138/files#diff-0bfee22e5235956a736cedde5812194226d112687d509d06c9295aa7679b7bb7R39-R78)